deployment/mcad-controller/templates/deployment.yaml

```diff
@@ -322,6 +322,18 @@ spec:
         imagePullPolicy: {{ .Values.httpServerImage.pullPolicy }}
         ports:
         - containerPort: 80
+        livesnessProbe:
+          httpGet:
+            path: /healthz
+            port: 80
+          initialDelaySeconds: 30
+          timeoutSeconds: 5
+        readinessProbe:
+          httpGet:
+            path: /readyz
+            port: 80
+          initialDelaySeconds: 30
+          timeoutSeconds: 5
       - name: "quota-management"
```

Comment:
Does the quota management controller expose health endpoints?

Reply:
I'm not quite sure where it would go, is it somewhere in [here](https://github.com/project-codeflare/multi-cluster-app-dispatcher/blob/2e817aea9aaf014de76b1898f9198629ea7a6128/pkg/quotaplugins/quota-forest/quota-manager/main.go#L28-L29)?

Reply:
Possibly, otherwise maybe probes for sidecar containers can be removed for now, and iterate once it's clearer what does health checking mean for these?